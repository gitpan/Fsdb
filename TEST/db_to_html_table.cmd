prog='db_to_html_table'
args='-g 3'
cmp='diff -cb '

