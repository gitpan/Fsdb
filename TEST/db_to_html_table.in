#fsdb -F S account passwd uid gid fullname homedir shell
johnh  *  2274  134  John & Ampersand  /home/johnh  /bin/bash
greg  *  2275  134  Greg < Lessthan  /home/greg  /bin/bash
root  *  0  0  Root ; Semi  /root  /bin/bash
four  *  1  1  Fourth Row  /home/four  /bin/bash
five  *  1  1  5th Row  /home/cinq  /bin/bash
six  *  1  1  6th Row  /home/seis  /bin/bash
seven  *  1  1  7th Row  /home/sieben  /bin/bash
