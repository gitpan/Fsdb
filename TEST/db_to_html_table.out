<table>
<tr><th>account</th> <th>passwd</th> <th>uid</th> <th>gid</th> <th>fullname</th> <th>homedir</th> <th>shell</th> </tr>
<tr bgcolor="#f0f0f0"><td>johnh</td> <td>*</td> <td>2274</td> <td>134</td> <td>John &amp; Ampersand</td> <td>/home/johnh</td> <td>/bin/bash</td> </tr>
<tr bgcolor="#f0f0f0"><td>greg</td> <td>*</td> <td>2275</td> <td>134</td> <td>Greg &lt; Lessthan</td> <td>/home/greg</td> <td>/bin/bash</td> </tr>
<tr bgcolor="#f0f0f0"><td>root</td> <td>*</td> <td>0</td> <td>0</td> <td>Root ; Semi</td> <td>/root</td> <td>/bin/bash</td> </tr>
<tr bgcolor="#d0d0d0"><td>four</td> <td>*</td> <td>1</td> <td>1</td> <td>Fourth Row</td> <td>/home/four</td> <td>/bin/bash</td> </tr>
<tr bgcolor="#d0d0d0"><td>five</td> <td>*</td> <td>1</td> <td>1</td> <td>5th Row</td> <td>/home/cinq</td> <td>/bin/bash</td> </tr>
<tr bgcolor="#d0d0d0"><td>six</td> <td>*</td> <td>1</td> <td>1</td> <td>6th Row</td> <td>/home/seis</td> <td>/bin/bash</td> </tr>
<tr bgcolor="#f0f0f0"><td>seven</td> <td>*</td> <td>1</td> <td>1</td> <td>7th Row</td> <td>/home/sieben</td> <td>/bin/bash</td> </tr>
</table>
